Makefile

```diff
@@ -140,3 +140,15 @@ deps/down: ## Delete containers dependencies.
 	@echo "Deleting dependencies "
 	@docker-compose --project-name maestro down
 	@echo "Dependencies deleted successfully."
+
+
+#-------------------------------------------------------------------------------
+#  Easily start Maestro
+#-------------------------------------------------------------------------------
+.PHONY: maestro/start
+maestro/start: build-linux-x86_64 ## Start Maestro with all of its dependencies
+	@echo "Starting maestro..."
+	@cd ./e2e/framework/maestro; docker-compose --project-name maestro up -d
+	@go run main.go migrate;
+	@cd ./e2e/framework/maestro; docker-compose --project-name maestro start
```

Comment:
Why do you run docker-compose `up` and `start`

Reply:
Because after you use `up`, the worker does not work since the migration hasn't been executed. Because of that, we use up, run migrate, then start everything.

Reply:
I improved the readability of it, can you look it again?